src/components/sharedLayout.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import Header from './header';
+import Footer from './footer';
+
+export const SharedLayout = ({ children, pkgVersion }) => {
+  return (
+    <>
+      <Header pkgVersion={pkgVersion} />
+      <main
+        className="bg-white text-gray-600 flex flex-1 flex-col scrolling-touch -mt-12 w-3/4 m-auto"
```

Comment:
Curious about the negative top margin `-mt-12` here - is that required?  If this is a shared layout, it seems like maybe not all layouts will want this negative top margin?